components/observations/uploader/ObservationsUploader.ts

```diff
@@ -197,6 +197,43 @@ export class ObservationUploader {
         });
       });
 
+      observationFormData.avalanches.forEach((avalanche, index) => {
+        avalanche.images?.forEach(({image, caption}) => {
+          tasks.push({
```

Comment:
If the shape of the task being pushed is 99% identical to the other one, a refactor to a common helper with a callout for the diff would be nice